src/Pages/GradeSelect/SearchWidget/index.js

```diff
@@ -0,0 +1,330 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styled from 'styled-components/macro';
+import TabPanel from './TabPanel';
+
+function SearchWidget(props) {
+  const [departure, setDeparture] = useState('');
+  const [arrival, setArrival] = useState('');
+  const [startDate, setStartDate] = useState(new Date());
+  const [endDate, setEndDate] = useState(undefined);
```

Comment:
```suggestion
  const [destination, setDestination] = useState({ departure: "", arrival: "" });
  const [date, setDate] = useState({ start: new Date(), end: undefined });
```
이런 느낌은 어떨까요? 상태 수도 줄이고 서로 연관된 데이터라는 걸 코드로 표현할 수 있을 것 같습니다

Reply:
선주님 되는 걸로 같이 맞춰보겠습니다